### DISCUSSION TITLE
draw circle conditionally in a cell

### DISCUSSION THREAD

**Discussion by ahmad-abusaa** [exceljs/exceljs#2493] [Q&A]
how can i draw a circle around number in a cell conditionally ?

**Comment by TranBaVinhSon** [comment]
Sample question! Do you have any solution ?

**Comment by TranBaVinhSon** [comment]
For example, I want to put a circle around `昭 5` or a circle inside the cell of `昭 5`, is that possible ?
<img width="167" alt="Screen Shot 2019-07-22 at 12 32 15" src="https://user-images.githubusercontent.com/12994069/61606269-63b55700-ac84-11e9-95de-ef0f65d494b3.png">
